workbench-core/accounts/src/services/projectService.ts

```diff
@@ -80,4 +80,21 @@ export default class ProjectService {
 
     return Promise.resolve({ data: projectsResponse.Items as unknown as Project[] });
   }
+
+  /**
+   * Check whether a CostCenter have any projects associated with it
+   * @param costCenterId - id of CostCenter we want to check
+   * @returns Whether a CostCenter have any projects associated with it
+   */
+  public async doesCostCenterHaveProjects(costCenterId: string): Promise<boolean> {
+    const queryParams: QueryParams = {
+      index: 'getResourceByDependency',
+      key: { name: 'resourceType', value: 'project' },
+      sortKey: 'dependency',
+      eq: { S: costCenterId }
+    };
+
+    const associatedProjResponse = await this._aws.helpers.ddb.query(queryParams).execute();
```

Comment:
You should use `getPaginatedItems()` instead of `query().execute()`

Reply:
updated